test/e2e/specs/all-features-confirmation/recordedit/add.spec.js

```diff
@@ -212,10 +221,43 @@ describe('Record Add', function() {
                 }
             });
         });
-
+        describe('Markdown Editor Help button is clicked, ', function() {
```

Comment:
Mention in a comment that the test cases inside this `describe` assume that,

1. We are in recordedit page.
2. There's at least a column with markdown type.